src/pages/Home/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import ArticleCard from '../../commons/ArticleCard'
+import Skeletons from '../../commons/Skeletons'
+import Title from '../../commons/Title'
+import Wrapper from '../../partials/Wrapper'
+import getArticles from '../../redux/actions/articles/getArticles'
+import Publishers from './Publishers'
+import { setFilterKeyWord } from '../../redux/features/articles'
+
+function Home() {
+    const dispatch = useDispatch();
+    const {data, loading} = useSelector(state => state.articles.articles);
+    const {data: filtered} = useSelector(state => state.articles.filtered);
+    const filterKeyword = useSelector(state => state.articles.filterKeyword);
+
+    useEffect(() =>{
+        dispatch(getArticles(12))
+    }, []);
+
+  return (
+    <Wrapper>
+        <div className='mt-14'>
+            <Title text={filterKeyword ? `Result for "${filterKeyword}" in articles`: "The world's latest news"} onBack={() =>dispatch(setFilterKeyWord(""))} withBackIcon={filterKeyword} />
+            <div className="grid sm:grid-cols-3 gap-10 mt-16">
+                {
+                    loading ?
+                    <Skeletons count={9} />:
```

Comment:
My design requirement (grid) uses any multiple of 3 elements, that's why I chose to use 9 skeletons since they're just there to show that the items are still being fetched but are not the representation of the items by themselves

Reply:
Got it.